.circleci/config.yml

```
@@ -64,7 +64,7 @@ jobs:
       - install_dependencies
       - run:
           name: Run eslint
-          command: yarn run eslint app/assets app/javascript    
+          command: yarn run eslint app/assets app/javascript --ext .js,.jsx,es6   
```

```suggestion
          command: yarn run eslint app/assets app/javascript --ext .js,.jsx,.es6   
```
I'm not sure that it matters. Also wonder if there's a way to add this to the config, so we get the right behavior by default. I'll file a separate issue for that.

---

Filed #955 for follow-up.